scripts/add_attribute.py

```diff
@@ -119,7 +119,7 @@
 
 added = False
 
-inInit = False
+inInit = True if line == "    def _initAttributes(self):" else False
```

Comment:
This is awkward. How about?

`inInit = line.endswith("def _initAttributes(self):")`

Reply:
@s-t-e-v-e-n-k , sure - updated.
What a pitty, there are so view chances to use ternary operators...

Reply:
Haha! I am not against them, but only when absolutely required :-)